tests/languages/nodejs/Makefile

```diff
@@ -0,0 +1,34 @@
+include ../../common.mk
+
+PROG=/app/index.js
+ENCLAVE_CMD=usr/local/bin/node ${PROG}
```

Comment:
Leading `/` missing?

Reply:
It was working since app runs in root folder. But added, it is better like /usr/...